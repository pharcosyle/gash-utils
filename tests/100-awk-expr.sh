awk -f tests/data/awk/expr.awk tests/data/awk/nfset.in
