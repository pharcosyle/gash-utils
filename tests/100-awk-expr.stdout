one
two
other
other
other
